lib/migration-chunks/validation/field.js

```diff
@@ -10,6 +10,152 @@ const invalidActionError = (message, step) => {
   };
 };
 
+const RELATIVE_MOVEMENTS = ['afterField', 'beforeField'];
+
+function fieldId (step) {
+  return step.payload.fieldId;
+}
+
+function pivotId (step) {
+  return step.payload.movement.pivot;
+}
+
+function contentTypeId (step) {
+  return step.payload.contentTypeId;
+}
+
+function fieldMovementKey (step) {
+  return [contentTypeId(step), fieldId(step)].join('/');
+}
+
+function fieldExists (validationContext, fieldId) {
+  return validationContext.fieldSet.has(fieldId);
+}
+
+function fieldHasBeenRemoved (validationContext, fieldId) {
+  return validationContext.fieldRemovals.has(fieldId);
+}
+
+function fieldHasBeenMoved (validationContext, movementKey) {
+  return validationContext.fieldMovements.has(movementKey);
+}
+
+const checks = {
+  'field/create' (errors, step, validationContext) {
```

Comment:
What was the decision now on this func-style? I find it quite confusing, but could live with it - we're not using this style anywhere else, though 🤔

Reply:
ah, that was me being fancy. Actually I don't know why I used it when I don't particularly like it. Will change it : )